WooCommerce/WooCommerceTests/ViewRelated/Dashboard/DashboardUIFactoryTests.swift

```diff
@@ -15,47 +15,49 @@ final class DashboardUIFactoryTests: XCTestCase {
         super.tearDown()
     }
 
-    func testWhenV4IsAvailableWhileNoStatsVersionIsSaved() {
+    func test_it_loads_only_statsV4_when_V4_is_available_while_no_stats_version_is_saved() {
```

Comment:
💯 I'd say to refactor also the other method names in this class 🗡️

Reply:
Sure! https://github.com/woocommerce/woocommerce-ios/pull/2832/commits/7c3f2343619a42dfcdce56aff5c1e47b257e0afb